.github/workflows/README.md

```diff
@@ -10,3 +10,30 @@ This command creates custom jars that we can share with customers.
 
 - **ref** - the branch/tag/hash that should be checked out to generate the jar;
 - **description** - a description that will show on the summary, so we know the contents of that custom jar.
+
+
+### Z - Sandbox job
+
```

Comment:
I'm fine with this but I don't have to push workflow to `main` to test my new workflows that I remember.

Reply:
I tried a few times to have a dispatcher show up in the list, but no luck.
I ended up overwriting the verify instrumentation to test my workflow, which I didn't like because it added a bunch of runs that were not related to that original job.

Reply:
I just made it run off updates to by dev branch and it shows up